android/app/src/main/java/it/teamdigitale/app/italiaapp/MainActivity.java

```diff
@@ -54,6 +55,12 @@ protected void onCreate(Bundle savedInstanceState) {
         if (android.os.Build.VERSION.SDK_INT != Build.VERSION_CODES.O) {
             setRequestedOrientation(ActivityInfo.SCREEN_ORIENTATION_PORTRAIT);
         }
+
+        // Prevent the screen content being visible when the app is in background
+        getWindow().setFlags(
```

Comment:
Could you check if this we already have this behavior? see here https://github.com/pagopa/io-app/pull/917

Reply:
You're right, it was disabled for the debug mode! I reverted the file BUT another control was required: on Android, ask for identification when the state returns to active makes the secured screen being displayed for a while before the Identification screen:
EXAMPLE:
![bug on visibility](https://user-images.githubusercontent.com/38431762/73745625-bf908a80-4753-11ea-9da8-8d9beb5fed0c.gif)

Dispatching the Identification request when we goes on background solves the issue.